src/cascadia/TerminalApp/GlobalAppSettings.cpp

```diff
@@ -43,26 +43,53 @@ static constexpr std::string_view SoftwareRenderingKey{ "experimental.rendering.
 static constexpr std::string_view ForceVTInputKey{ "experimental.input.forceVT" };
 
 // Launch mode values
-static constexpr std::wstring_view DefaultLaunchModeValue{ L"default" };
-static constexpr std::wstring_view MaximizedLaunchModeValue{ L"maximized" };
-static constexpr std::wstring_view FullscreenLaunchModeValue{ L"fullscreen" };
+static constexpr std::string_view DefaultLaunchModeValue{ "default" };
+static constexpr std::string_view MaximizedLaunchModeValue{ "maximized" };
+static constexpr std::string_view FullscreenLaunchModeValue{ "fullscreen" };
 
 // Tab Width Mode values
-static constexpr std::wstring_view EqualTabWidthModeValue{ L"equal" };
-static constexpr std::wstring_view TitleLengthTabWidthModeValue{ L"titleLength" };
-static constexpr std::wstring_view TitleLengthCompactModeValue{ L"compact" };
+static constexpr std::string_view EqualTabWidthModeValue{ "equal" };
+static constexpr std::string_view TitleLengthTabWidthModeValue{ "titleLength" };
+static constexpr std::string_view TitleLengthCompactModeValue{ "compact" };
 
 // Theme values
-static constexpr std::wstring_view LightThemeValue{ L"light" };
-static constexpr std::wstring_view DarkThemeValue{ L"dark" };
-static constexpr std::wstring_view SystemThemeValue{ L"system" };
+static constexpr std::string_view LightThemeValue{ "light" };
+static constexpr std::string_view DarkThemeValue{ "dark" };
+static constexpr std::string_view SystemThemeValue{ "system" };

```

Comment:
all these wide strings going to narrow will save 1. binary space and 2. conversion time and kernel round trips

Reply:
_thank you_. I was always confused why some were `wstring_view` and others were `string_view` haha